src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -4707,8 +4707,8 @@ namespace HeatBalanceManager {
         // This subroutine only gets called once in the GetConstructionData subroutine
 
         // ASHRAE Handbook Fundamental 2005
-        // Thermal resistance of the inside air film, m2.K/W. Average of 0.14 (heat flow up) and 0.11 (heat flow down)
-        Real64 constexpr Rfilm_in(0.125);
+        // Thermal resistance of the inside air film, m2.K/W. Average of 0.16 (heat flow down) and 0.11 (heat flow up)
+        Real64 constexpr Rfilm_in(0.135);
```

Comment:
A simple change to update film resistance to match ASHRAE Fundamentals data and also used elsewhere in E+ code.

Reply:
The only question I would have is if I should have used more precision with (0.1620 + 0.1074)/2 = 0.1347. Diffs I've seen are very small so maybe using existing precision is sufficient.